Sources/StreamChat/WebSocketClient/WebSocketConnectPayload.swift

```diff
@@ -4,24 +4,34 @@
 
 import Foundation
 
-struct WebSocketConnectPayload: Encodable {
+struct WebSocketConnectPayload<ExtraData: ExtraDataTypes>: Encodable {
     private enum CodingKeys: String, CodingKey {
         case userId = "user_id"
         case userDetails = "user_details"
         case serverDeterminesConnectionId = "server_determines_connection_id"
     }
     
     let userId: UserId
-    let userDetails: UserWebSocketPayload
+    let userDetails: UserWebSocketPayload<ExtraData>
```

Comment:
why do we send user id twice? 🤔It's in `WebSocketConnectPayload.userId` and in `WebSocketConnectPayload.userDetails.id`

Reply:
I'm not sure, just preserved this behaviour. Maybe Vojta remembers

Reply:
ok 👍 not a blocker for me but would be nice to ask backend team